api/app/controllers/webhook.controller.js

```diff
@@ -0,0 +1,48 @@
+const db = require("../sequelize/models");
+const User = db.user;
+const Review = db.review;
+const Transcript = db.transcript;
+const Op = db.Sequelize.Op;
+
+
+exports.create = async (req, res) => {
+    // const action = req.headers["x-github-event"];
+    const pull_request = req.body;
+     const action = pull_request.action;
+
+
+    // Check if it's a pull request event
+    if (action === 'opened' || action === 'closed' || action === 'synchronize') {
+        const html_url = pull_request.pull_request.html_url;
+
+        console.log(`Received pull request event for PR: ${html_url}`);
+
+        try {
+            // Check if the PR URL exists in the database
+            const existingReview = await Transcript.findOne({ where: { pr_url: html_url } });
+
+            if (existingReview) {
+                // PR is merged, update the mergeAt timestamp
+                existingReview.mergedAt = new Date();
```

Comment:
Also, I think we should handle them differently.
- If it is merged, fill in the `mergedAt` timestamp.
- If it was closed("not merged") fill in the `archivedAt` time stamp